### DISCUSSION TITLE
Repeater only working when changing network name

### DISCUSSION THREAD

**Discussion by gselsidi** [Mygod/VPNHotspot#346] [Q&A]
Xioami Note 9 Pro
Android 11 RKQ1
Miui 12.5.2

1. Click Repeater just stays grayed out and doesn't start
2. Click Wifi Config top Right
3. Click Repeater
4. Change network name
5. Click repeater again

I have to change the network name for repeater to start properly, or it never starts, other times it wouldn't even let me go into the settings to change the network name.

logs attached 
[logs.log](https://github.com/Mygod/VPNHotspot/files/8188723/logs.log)

**Comment by Mygod** [comment]
Seems like a ROM issue. Report to MIUI instead.
